### DISCUSSION TITLE
Unable to sing in: Failed sending HTTP request.

### DISCUSSION THREAD

**Discussion by digital564** [jlesage/docker-crashplan-pro#472] [General]
![image](https://github.com/user-attachments/assets/bd810179-9b4f-4f0e-bdce-2bdf4c43f3f3)

After setting up my synology container I get this error when login with the Authentication code:

Unable to sing in: Failed sending HTTP request.

I tried to copy the URL and log from the browser and it is working but not inside the docker.

**Comment by itsik259** [comment]
I have the exact same problem. 
Any ideas anyone?

**Reply by digital564** [reply]
I went to my crashplan account and gave permissions to my user

**Reply by CreakyLid** [reply]
> I went to my crashplan account and gave permissions to my user

Can you remember which permissions it was that you gave? Or where these permissions were or how you did it?

I believe that I have correctly added every single permission available, however I still get this error?

![image](https://github.com/user-attachments/assets/097ebcfb-990c-4241-8c23-5cb125fbb9d0)

I can paste the sign-in link into a second browser tab, and when I go back to the CrashPlan tab I am successfully signed in; so that works at least.

I am running the Unraid community application docker template of CrashPlanPRO fyi

**Comment by DBtheDBA** [comment]
Having the exact same issue.  Anyone find a resolution to this?

Began experiencing this same issue after restarting my container on 4 days ago.  I can't re-login to the server after it scrubbed my keys due to the following (a guess based upon the log):
`[04.07.25 20:55:27.471 INFO  main         om.backup42.service.IdentityInfo] CHANGE COMPUTER IDENTITY! GUID reset but not changing.`
I rebooted the host to apply some standard updates, nothing else changed.

I'm unsure why my container restart caused this behavior, but now I can't get my container reauthed. With effectively some of the same behaviors as above.  I've opened a ticket with Crashplan, but haven't received a response yet.
`[04.10.25 23:32:57.877 INFO  34077_AUTH-1   com.backup42.service.CPService] DEAUTHORIZE FINISHED`
`[04.10.25 23:32:57.880 INFO  34077_AUTH-1 backup42.service.peer.Authorizer] AUTH:: Error = CPErrors.Global.SYSTEM : []`

I could chose to scrap my persistent volume and restart as an option listed in another bug, but wouldn't that effectively then need me to start the backup from scratch? I really don't know what triggered all of this which in turn makes me skeptical of what the fix should be.

**Reply by DBtheDBA** [reply]
Yeah, still investigating.  Something goofy has changed on the filesystem side.  Still debugging be unsure why these issues would crop up out of nowhere:
`[04.12.25 12:18:07.940 INFO  77472_AUTH-1 ashplan.io.FilePermissionService] FilePermission:: Permissions for /usr/local/crashplan/cache: [OWNER_RE
AD, OWNER_WRITE, OWNER_EXECUTE, GROUP_READ, GROUP_WRITE, GROUP_EXECUTE]
[04.12.25 12:18:07.942 WARN  77472_AUTH-1 vice.config.ServiceConfigService] SC:: Failed to initialize config, using default., java.nio.file.FileSy
stemException: /usr/local/crashplan/manifest: Operation not permitted
STACKTRACE:: java.nio.file.FileSystemException: /usr/local/crashplan/manifest: Operation not permitted`

**Reply by DBtheDBA** [reply]
Ended up just rebuilding the static volume per suggestion [here](https://github.com/jlesage/docker-crashplan-pro/issues/449), I likely spent too many hours trying to chase the root cause, but it's working now and resyncing as a "replaced device".  Can't see anything specific to the permissions or files that are a potential cause.

**Reply by DBtheDBA** [reply]
Nevermind, this nightmare isn't over yet.  Now having issues with crashplan with undocumented IP ranges....more to come.

**Comment by digital564** [comment]
I'm moving away from crashplan, the app is too complicated, I don't know how to restore files when I need them. The app is constantly failing and the company is not transparent about the privacy of the data and file retention.

I recommend you guys to do your research.
